src/Entity/Activity.php

```diff
@@ -176,37 +178,47 @@ public function getStatus(): ?string
         return $this->status;
     }
 
+    public function getAllStatuses(): array
```

Comment:
This kind of functions should be static so you won't need an instance of the class to retrieve the values. Please make it static and read more about it.